completions/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+cmake_minimum_required(VERSION 3.10)
+project(quick-lint-js-completions)
+
+install(
+  FILES _quick-lint-js
+  DESTINATION "/usr/share/zsh/site-functions"
```

Comment:
Blocking: We should use `${DATAROOTDIR}` from `GNUInstallDirs` instead of hard-coding `/usr/share`. https://cmake.org/cmake/help/v3.10/module/GNUInstallDirs.html

See docs/CMakeLists.txt for a similar example.

Reply:
`Implemented`